docs/reference/search/profile.asciidoc

```diff
@@ -1272,6 +1272,7 @@ One of the `dfs.knn` sections for a shard looks like the following:
 "dfs" : {
     "knn" : [
         {
+        "vector_operations_count" : 4,
```

Comment:
Would it make sense to add a line in this document about what this represents? 
As a nitpick it's not immediately obvious to me that the name `vector_operations_count` means top K total hits.

Reply:
Updated